src/global.js

```diff
@@ -70,12 +70,18 @@ const conventionalCommitType = (types => {
 /**
  * Global options/settings. One time _set, then freeze.
  *
- * @type {{_set: *}}
+ * @type {{contextPath: string, _set: *}}
  */
 const OPTIONS = {
   contextPath,
   set _set(obj) {
-    Object.entries(obj).forEach(([key, value]) => (this[key] = value));
+    Object.entries(obj).forEach(([key, value]) => {
+      this[key] = value;
+
+      if (typeof value === 'function') {
```

Comment:
required:
move the condition up, we're currently running both... still works just clean up